bin/client_init.sh

```diff
@@ -65,7 +65,7 @@ ip addr
 ip route
 
 # Check we can connect to the GATEWAY IP
-ping -c "${CONNECTION_RETRY_COUNT}" "$GATEWAY_IP"
+ping -c "${CONNECTION_RETRY_COUNT}" "$GATEWAY_IP" || true
```

Comment:
Thanks for the PR

Could we keep the check? Perhaps by allowing ICMP in iptables

Reply:
This does work with Cilium's kube-proxy replacement aka the ping does work, so I'm not sue where you're running into issues?